src/App.tsx

```diff
@@ -1,61 +1,94 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+export const App: React.FC = () => {
+  const [currentTitle, setCurrentTitle] = useState('');
+  const [currentUserId, setCurrentUserId] = useState(0);
+  const [todos, setTodos] = useState(todosFromServer);
+  const [wasSubmitted, setWasSubmitted] = useState(false);
+
+  const isFormValid = currentTitle.trim() && currentUserId;
+
+  const formReset = () => {
+    setCurrentTitle('');
+    setCurrentUserId(0);
+  };
+
+  const onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (isFormValid) {
+      const newTodo = {
+        id: Math.max(...todos.map(todo => todo.id)) + 1,
```

Comment:
Ensure that the `todos` array is not empty before using `Math.max` to calculate the new ID. If `todos` is empty, `Math.max` will return `-Infinity`, which will cause an error. Consider using a default value like `0` in such cases.